tests/config/test_data.config

```diff
@@ -263,6 +263,9 @@ params {
             'txt' {
                 hello = "${test_data_dir}/generic/txt/hello.txt"
             }
+            'cnn' {
+                 reference = "${test_data_dir}/generic/cnn/reference.cnn"
```

Comment:
is there a corresponding PR on test-data? Can review that one as well then

Reply:
No, actually I just pushed it to the test-data repo. Doesn't seem to need code-review. That's why the unit tests are working already.